releases/release-1.21/release-notes/maps/pr-99068-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 99068
+releasenote:
+  text: |-
+    Unable read directory `/root/bin` from your PATH: open `/root/bin`: no such file or directory. Skipping...
+    error: unable to find any kubectl plugins in your PATH
```

Comment:
![image](https://media.giphy.com/media/3jN3GziOKUEmI/giphy.gif)

I think this is a case of no release note is needed :smile:
```suggestion
releasenote:
  do_not_publish: true
```

Reply:
Yes, it is a trivial change. Will remove it